src/config.rs

```diff
@@ -94,12 +95,22 @@ pub struct MapListenerConfig {
     pub threshold: i8,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Display)]
+pub enum TeleopMode {
+    // Node will keep speed button press only increment decrement
+    Classic,
+    // Speed will be reduced every iteration by one increment
+    // To keep driving a button needs to be pressed every increment
+    Safe,
+}
+
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct TeleopConfig {
     pub default_increment: f64,
     pub increment_step: f64,
     pub cmd_vel_topic: String,
     pub publish_cmd_vel_when_idle: bool,
+    pub mode: TeleopMode,
```

Comment:
Is this compatible with old config files? If not, derive Default for `TeleopMode` and mark it as `#[serde(default)]` here.